internal/app/machined/pkg/controllers/network/link_status_test.go

```diff
@@ -126,6 +128,59 @@ func (suite *LinkStatusSuite) assertNoInterface(id string) error {
 }
 
 func (suite *LinkStatusSuite) TestLoopbackInterface() {
```

Comment:
feels like test name should be swapped with the next test?

Reply:
lol, not sure how that happened 🙃